src/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -577,30 +590,64 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
                         allo.Payment__c = sObj.Id;
                     }
 
+                    if (allo.Percent__c != null) {
+                        allo.Amount__c = (amount * allo.Percent__c * .01).setScale(2);
+                        
+                    }
+
+                    //Calculate proportional amount for non-percentage based Payment allocations
+                    if (sObj instanceof npe01__OppPayment__c && allo.Percent__c == null) {
+                        allo.Percent__c = allo.Amount__c / wrap.parentAmount * 100;
```

Comment:
I believe it should not calculate the Percentage for the Default GAU.  If you have to do it here, for some reason, we will need to make sure we remove the percentage for the default.  I looked the code over again, but it didn't seem like it was removing the Default GAU from the listAlloForInsert.

Reply:
This doesn't calculate the percentage for the Default GAU. The Default GAU is not included in the listAlloForInsert. The listAlloForInsert is a clone of the list of Allocations in the mapWrapper, which excludes the default GAU when being populated in the "fillMapWrapper".